jest.config.js

```diff
@@ -0,0 +1,194 @@
+// For a detailed explanation regarding each configuration property, visit:
+// https://jestjs.io/docs/en/configuration.html
+
+module.exports = {
+	// All imported modules in your tests should be mocked automatically
+	// automock: false,
+
+	// Stop running tests after `n` failures
+	// bail: 0,
+
+	// Respect "browser" field in package.json when resolving modules
+	// browser: false,
```

Comment:
This should be set to true because this component is aimed for browser environments and when `occ` (origami component converter) has executed, it adds a `browser` field to the `package.json`.
```suggestion
	browser: true,
```